src/utils/draftbots.js

```diff
@@ -77,135 +79,156 @@ export const fetchLands = {
+export const getInternalSynergy = (combination, picked, synergies) => {
   if (picked.cards.length === 0 || !synergies) {
     return 0;
   }
 
-  let synergy = 0.000001;
-  let internalSynergy = 0.000001;
+  let internalSynergy = 0;
   if (synergies) {
     const pickedInCombo = picked.cards.filter((card2) => considerInCombination(combination, card2));
-    let count = 0;
     for (let i = 1; i < pickedInCombo.length; i++) {
       for (let j = 0; j < i; j++) {
-        internalSynergy += similarity(synergies[pickedInCombo[i].index], synergies[pickedInCombo[j].index]) ** 10;
-        count += 1;
+        const similarityValue = similarity(synergies[pickedInCombo[i].index], synergies[pickedInCombo[j].index]);
+        internalSynergy += -Math.log(1 - scaleSimilarity(similarityValue)) / SYNERGY_SCALE;
       }
     }
-    if (count) {
-      internalSynergy /= count;
-    }
-    if (card) {
-      for (const { index } of pickedInCombo) {
-        synergy += similarity(synergies[index], synergies[card.index]) ** 10;
-      }
-      if (pickedInCombo.length) {
-        synergy /= pickedInCombo.length;
+  }
+
+  const numPairs = (picked.cards.length * (picked.cards.length + 1)) / 2;
+  return internalSynergy / numPairs;
+};
+
+// How much does the card we're considering synergize with the cards we've picked?
+// Scale is roughly 0-1.
+export const getPickSynergy = (combination, card, picked, synergies) => {
+  if (picked.cards.length === 0 || !synergies) {
+    return 0;
+  }
+
+  let synergy = 0;
+  if (synergies && card) {
+    const pickedInCombo = picked.cards.filter((card2) => considerInCombination(combination, card2));
+    for (const { index } of pickedInCombo) {
+      // Don't count synergy for duplicate cards.
+      // Maximum synergy is generally around .997 which corresponds to ~1.
+      if (index !== card.index) {
+        const similarityValue = similarity(synergies[index], synergies[card.index]);
+        synergy += -Math.log(1 - scaleSimilarity(similarityValue)) / SYNERGY_SCALE;
       }
     }
   }
-  return Math.log(internalSynergy + synergy) + 13;
+  return synergy / picked.cards.length;
 };
 
+// Has this color combination been flowing openly?
+// Scale from 0-1.
 export const getOpenness = (combination, seen) => {
   if (seen.cards.length === 0) {
     return 0;
   }
 
-  const seenCount = COLOR_SCALING_FACTOR[combination.length] * seen[combination.join('')];
-  return Math.log(seenCount);
+  return seen[combination.join('')] / seen.cards.length;
 };
 
-export const getColor = (combination, picked, card) => {
-  return Math.log(
-    COLOR_SCALING_FACTOR[combination.length] * (picked[combination.join('')] + 10 ** ((card?.rating ?? 0) / 400)),
-  );
+// How good are the cards we've already picked in this color combo?
+// Scale from 0-1.
+export const getColor = (combination, picked) => {
+  if (picked.cards.length === 0) {
+    return 0;
+  }
+
+  return picked[combination.join('')] / picked.cards.length;
```

Comment:
This absolutely can't be an average, we need to commit more to a color the more we take of it.